### PR TITLE
ASC-1040 Refactor Test Case with Steps

### PR DESCRIPTION
**Pull request by ryan-rs**
Update the TestVerifyMBUInstall test case to use the new test case with steps
workflow provided by ASC tooling.

### CODE REVIEW

File: molecule/default/tests/test_verify_mbu_installed.py [hunk 1]
```diff
@@ -10,15 +16,41 @@
 backup_folder_on_container = '/var/backup'
 
 
-@pytest.mark.incremental
+# ======================================================================================================================
+# Helpers
+# ======================================================================================================================
+def run_on_container(command, container, run_on_host):
```
Comment (johnduarte) [inline molecule/default/tests/test_verify_mbu_installed.py]: Should this helper be removed and the one from https://github.com/rcbops/pytest-rpc/blob/master/pytest_rpc/helpers.py#L510-L527 ?

File: molecule/default/tests/test_verify_mbu_installed.py [hunk 2]
```diff
@@ -10,15 +16,41 @@
...
+    assert output.rc == 0
+    return output
+
+
+# ======================================================================================================================
+# Test Cases
+# ======================================================================================================================
+@pytest.mark.test_id('406be222-cbee-11e8-ae3a-0025227c8120')
+@pytest.mark.jira('asc-262')
+@pytest.mark.test_case_with_steps()
```
Comment (johnduarte) [inline molecule/default/tests/test_verify_mbu_installed.py:25]: With the exception of this addition, are the other changes in this file relevant to using steps or should they be broken out into a separate commit?

**Review by johnduarte — CHANGES_REQUESTED** [review]

File: molecule/default/tests/test_verify_mbu_installed.py (hunk 2 above)
Comment (ryan-rs) [reply molecule/default/tests/test_verify_mbu_installed.py:25]: @johnduarte Split PR into multiple commits to address your feedback.

**Review by johnduarte — APPROVED** [review]

**Review by phongdly — APPROVED** [review]
